.github/CODEOWNERS

```diff
@@ -674,6 +674,12 @@
 # ServiceLabel: %Monitor - LiveMetrics
 # ServiceOwners:                                                   @rajkumar-rangaraj @TimothyMothra @xiang17
 
+# PRLabel: %Monitor - Monitor
+/sdk/monitor/Azure.ResourceManager.Monitor/                        @thomasp98296 @ritwik8119
```

Comment:
Please move this down into the management block that starts around L915 and follow the patterns for management libraries there.   Please be sure to preserve alphabetical order by tag in that section.